src/components/Auth/AuthForm.spec.js

```diff
@@ -41,12 +41,19 @@ describe('AuthForm', () => {
 
 describe('Signup', () => {
   it('Register a new user on the signup form', async () => {
-    const { getByText, getByLabelText } = render(
+    const { getByText, getByTestId } = render(
       <BrowserRouter>
         <SignUpForm />
       </BrowserRouter>
     );
 
+    const form = getByTestId(/signupForm/i);
+    const usernameEle = form.querySelectorAll('input[name="username"]')[0];
```

Comment:
Just to avoid using id(`<input id='' .../>`) into input field! I tried to add `data-testid` inside TextField as well but data-testid was being applied to root element instead input. I can apply `getByLabelText` back if you want :)

Reply:
`getByLabelText` does not look for an ID attribute on the elements. It looks for a label, then the input field that it's tied to: https://testing-library.com/docs/dom-testing-library/api-queries#bylabeltext. I think we should prefer to use that instead in order to keep our testing pattern concise.

Reply:
Yeah, theoretically both approaches do the same thing, but I agree `getByLabelText` is cleaner, if you don't mind putting it back :)

Reply:
Done: https://github.com/codebuddies/frontend/pull/105/commits/972199515e83c879b76e76d485d08d0efc5638f6

Reply:
Thank you! I feel like an idiot -- of course we needed the `id="password"` on each of the `<Field>` components because that's partly how the label knows which input tag it is attached to. Does 9721995 look good to you too, @angelocordon?

Reply:
Looks good to me!